backend/models.py

```diff
@@ -218,11 +265,20 @@ class Shot(db.Model, AsDictMixin):
     yCord = db.Column(db.Float, nullable=True)
     yCordAdjusted = db.Column(db.Float, nullable=True)
 
-    # note still needs to the POIU table references.
-
-
     def __repr__(self):
         return F"<Shot {self.id}>"
+    
+
+class ShotPOIU (db.Model, AsDictMixin):
+    shot_poiu_id = db.Column(db.Float, primary_key=True, autoincrement=True)
+    id = db.Column(db.Float, nullable=True)
+    shooting_poiu_id = db.Column(db.Integer, nullable=True)
+    defending_poiu_id = db.Column(db.Integer, nullable=True)
+    
+    
+    def __repr__(self):
+        return F"<ShotPOIU {self.id}>"
```

Comment:
Once we merge this we'll have to update all of the API endpoints that are fetching this data.

The following need to be updated (not necessarily by you)
server.py:
- get_units_by_player_id
- get_shots_by_poiu 

Note these are currently in a PR.